src/drivers/boards/mavstation/module.mk

```diff
@@ -0,0 +1,16 @@
+#
+# Board-specific startup code for the mavstation
+#
+
+SRCS		= mavstation_init.c \
+		  mavstation_pwm_servo.c \
+		  mavstation_usb.c \
+		  mavstation_led.c
+
+ENABLE_CXXINITIALIZE=$(call check_nuttx_config ,"CONFIG_HAVE_CXX 1", $(NUTTX_CONFIG_H))
+ENABLE_CXXINITIALIZE+=$(call check_nuttx_config ,"CONFIG_HAVE_CXXINITIALIZE 1", $(NUTTX_CONFIG_H))
+ifeq ("$(ENABLE_CXXINITIALIZE)",$(nuttx_config_2true))
+SRCS		 +=	   ../../../modules/systemlib/up_cxxinitialize.c
```

Comment:
It doesn't not work... I had issues with this earlier but this snippet seems to work.  I didn't see the changes you made to revert back from this for the other configurations.  I can update the format based on the other configurations based on your commits from 5 days ago.